packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -16,15 +16,13 @@ export class MongoDBDataProvider<Type = any> implements GraphbackDataProvider<Ty
   protected collectionName: string;
   protected tableMap: ModelTableMap;
   protected fieldTransformMap: FieldTransformMap;
-  protected coerceTSFields: boolean;
 
   public constructor(model: ModelDefinition, db: any) {
     this.verifyMongoDBPrimaryKey(model.graphqlType.name, model.primaryKey);
     this.db = db;
     this.tableMap = buildModelTableMap(model.graphqlType);
     this.collectionName = this.tableMap.tableName;
     this.fieldTransformMap = getFieldTransformations(model.graphqlType);
-    this.coerceTSFields = parseMetadata("versioned", model.graphqlType);
```

Comment:
I suppose this was not needed anymore since we are using `GraphbackTimestamp` scalar? Good catch on that one.

Reply:
Correct!

I removed the code from the `queryBuilder` that corresponded to this flag:

https://github.com/aerogear/graphback/blob/master/packages/graphback-runtime-mongodb/src/queryBuilder.ts#L81-L87

The function description says it is a workaround until timestamp support was added.

Reply:
Nice change. Thanks!